metabase/metabase
# Add firewall info block when connecting to a database on Metabase Cloud

Epic: https://github.com/metabase/metabase/issues/18670

Adds a block with a list of Cloud IP addresses to database forms.

<img width="707" alt="Screen Shot 2021-10-26 at 5 23 35 pm" src="https://user-images.githubusercontent.com/8542534/138899188-479a0f8b-803f-41b7-b9e8-063f074f4ad1.png">

How to test:
- Change MetabaseSettings to `isHosted()` return `true`
- Change MetabaseSettings to `cloudGatewayIps` return `[]`
- Make sure that there is no Cloud IP address block in database forms
- Change MetabaseSettings to `cloudGatewayIps` return `["1.1.1.1"]`
- Make sure that there is a Cloud IP address block in database forms
